src/js/components/Select/Select.js

```diff
@@ -152,9 +152,33 @@ const Select = forwardRef(
       if (onClose) onClose();
     }, [onClose]);
 
+    const triggerChangeEvent = useCallback(
+      nextValue => {
+        // Calling set value function directly on input because React library
+        // overrides setter `event.target.value =` and loses original event
+        // target fidelity.
+        // https://stackoverflow.com/a/46012210
+        const nativeInputValueSetter = Object.getOwnPropertyDescriptor(
+          window.HTMLInputElement.prototype,
+          'value',
+        ).set;
+        nativeInputValueSetter.call(inputRef.current, nextValue);
+        const event = new Event('input', { bubbles: true });
+        inputRef.current.dispatchEvent(event);
+      },
+      [inputRef],
```

Comment:
Is `inputRef` really needed, since the `current` property is what can change.

Reply:
You're right, I can remove `inputRef` and it doesn't flag any missing dependencies.